CMake/cdat_modules_extra/git_clone.sh.in

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 cd @CMAKE_INSTALL_PREFIX@
-@GIT_EXECUTABLE@ clone --depth 1 -b @BRANCH@ @GIT_URL@ @GIT_TARGET@
+"@GIT_EXECUTABLE@" clone --depth 1 -b @BRANCH@ @GIT_URL@ @GIT_TARGET@
```

Comment:
@aashish24 should we split this into 2 lines for RH and older system like we did for Paraview?
I know it will make it bigger but we can clean it before tarring?
```
"@GIT_EXECUTABLE@" clone  @GIT_URL@ @GIT_TARGET@
"@GIT_EXECUTABLE@" checkout @BRANCH@
```